macros/edr/dbt_artifacts/upload_dbt_models.sql

```diff
@@ -71,8 +77,9 @@
         'package_name': node_dict.get('package_name'),
         'original_path': node_dict.get('original_file_path'),
         'path': node_dict.get('path'),
-        'generated_at': elementary.datetime_now_utc_as_string()
-    }%}
+        'generated_at': elementary.datetime_now_utc_as_string(),
+        'bigquery_partition_by_field': config_dict.get("partition_by", {}).get("field"),
```

Comment:
Maybe get the entire "partition_by" json?
I think the type and granularity may also be useful